### ISSUE TITLE
Failing to load pre-trained set

### ISSUE THREAD

**Issue by jberta1** [XHUJOY/CycleGAN-tensorflow#35]
Hi,

After downloading the pre-trained set and untar-ing to the a 'checkpoint' folder, I try  test run.
I get:
"[...]
discriminatorA/d_h3_pred/Conv/weights:0
 [*] Reading checkpoint...
 [!] Load failed...
Processing image: ./datasets/horse2zebra/testA\n02381460_1000.jpg
"

DIfficult to tell why the model won't load.
I'm on WIN10 if that matters, any advice wellcome
Thanks

Julien

**Comment by patakk** [comment]
I had the same problem. The solution is to run the test with the exact same parameters as when you trained the model. For example, if you used a different learning rate than the default one, you should run the test with the same different learning rate argument.
Hope that helps.

**Comment by meggiezhou** [comment]
I have the same question , and  i don't change any parameters, what should i do to solve it

**Comment by shrutiramesh1988** [comment]
I'm facing same issue after using the pre-trained model on linux with TF 1.15 + MKLDNN :
 
discriminatorA/d_bn3/offset:0
discriminatorA/d_h3_pred/Conv/weights:0
[*] Reading checkpoint...
 [!] Load failed...

I do not want to train the model, as I'm interested in inference only. How do we solve this issue?
